cmd/up.go

```diff
@@ -444,8 +446,9 @@ func (cmd *UpCmd) devPodUpProxy(
 
 		// run devpod up elsewhere
 		err := client.Up(ctx, client2.UpOptions{
-			CLIOptions: baseOptions,
-			Debug:      cmd.Debug,
+			CLIOptions:      baseOptions,
+			Debug:           cmd.Debug,
+			SkipNonBlocking: cmd.SkipNonBlocking,
```

Comment:
Can you move it into `CLIOptions` please? This way you don't have to define it on every command